src/parse/asp/objects.go

```diff
@@ -822,7 +822,7 @@ type pyConfig struct {
 
 func (c *pyConfig) MarshalJSON() ([]byte, error) {
 	if c.overlay == nil {
-		return json.Marshal(c.overlay)
```

Comment:
this was possibly a bug? Looks a bit sus...

Reply:
Yeah almost certainly. nice spot!